SYCL/Matrix/joint_matrix_ss_int8.cpp

```diff
@@ -11,6 +11,8 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 
+// XFAIL: *
+
```

Comment:
If this is just always going to fail, is there any plan to fix or support it?  If so, it would be good to mention it in a comment, otherwise someone may just remove this test.

Reply:
I added xfail because the test will fail until two internal patches that are currently under review get merged.
Once all the patches are in, I will remove xfail.
Should I add a comment about it or remove xfail?